src/irr/asset/CCPUMeshPacker.h

```diff
@@ -25,9 +25,21 @@ class CCPUMeshPacker final : public IMeshPacker<ICPUMeshBuffer, MDIStructType>
 
 	template <typename Iterator>
 	ReservedAllocationMeshBuffers alloc(const Iterator begin, const Iterator end);
-	virtual void commit() override;
 
-	ICPUBuffer* getMultiDrawIndirectBuffer() { return m_outMDIData.get(); }
+	//needs to be called before first commit
+	void createOutputBuffers();
```

Comment:
rename to `instantiateDataStorage` also take a `shrink_address_space_size_target=~0u` argument